client/contracts/truffle/contracts/oracle/OracleMgr.sol

```diff
@@ -55,7 +57,15 @@ contract OracleMgr is Pausable {
         require(_price > 0);
         _;
     }
-
+    /**
+     * Constructor.
+     * @param _initialPrice Initial Price.
+     * @param _baseDeposit base deposit for Oracle.
+     * @param _maxNumOracles Maximum numbers of Oracles.
+     * @param _freezePeriod Freeze period for Oracle.
```

Comment:
Oracle's deposits

Reply:
FreezePeriod is "Oracle's deposits"? Or you meant something different?